templates/project-listing.html

```diff
@@ -0,0 +1,44 @@
+{% set current_page = request.path.raw.split('/')[1] %}
+{% for i, project in enumerate(project_list, start=1) %}
+    <tr>
+        <td class="item">
+            <img src="{{ project.get_image_url('small') }}">
+            <a class="name" href="/{{ project.slug }}/">{{ project.name }}</a>
+
+            <div class="details{% if project.status == 'approved' %} room-for-numbers{% endif %}{% if current_page == '~' %} profile{% endif %}">
```

Comment:
E.g., this needs to check for user.participant or user.ADMIN as well as current_page.